app/src/main/java/io/github/droidkaigi/confsched2018/presentation/common/pref/Prefs.kt

```diff
@@ -18,4 +19,23 @@ object Prefs : KotprefModel() {
             context.bool(R.bool.pref_default_value_enable_hide_bottom_navigation),
             R.string.pref_key_enable_hide_bottom_navigation
     )
+    var enableReopenPreviousRoomSessions: Boolean by booleanPref(
+            context.bool(R.bool.pref_default_value_enable_reopen_previous_room_sessions),
+            R.string.pref_key_enable_reopen_previous_room_sessions
+    )
+    var previousSessionTab: String by stringPref(
+            context.getString(R.string.pref_default_value_previous_session_tab),
+            R.string.pref_key_previous_session_tab
```

Comment:
I think if we do not use Preference Screen, we don't have to use pref key. 👀 
Variable name will become key by Kotpref

Reply:
Oh...Thanks. 
I see, I will fix it.

Reply:
Can you use the enum support? 👀 
"com.chibatching.kotpref:enum-support:2.3.0"
https://github.com/chibatching/Kotpref#declare-preference-model